spotify.md

```diff
@@ -0,0 +1,118 @@
+# 10 tips for maintaining a long-term relationship with docs like code
+
+##### By Gary Niemen, Product Manager for TechDocs, Spotify’s docs-like-code solution
+
+I remember about two years ago sitting here (well, not precisely “here” because we sat in offices in those days) watching, wide-eyed and a bit confused, a video of [Riona MacNamara’s talk from Write the Docs 2015](https://youtu.be/EnB8GtPuauw). In the talk, Riona described how she and another technical writer used a docs-like-code approach to change Google engineering culture around technical documentation.
+
+I remember seeing this adoption curve and saying to myself: We need to do this. I want that curve.
+
+![image](./images/adoption1.png)
+
+And just a few months later, we had built and released TechDocs, Spotify’s docs-like-code solution. And a few months after that, we had the curve that I had longed for. Nice gradient, eh?
+
```

Comment:
Hmm I've read the blog post that you link to at the end of the article... originally I was going to say "link to TechDocs here" but now I see it's more complex than that. Still, I think you may want to have a list of links somewhere in the post as a "Resource" collection perhaps?

Reply:
I am happy for you to add that resources list. Use the links in the blog post under the heading: Okay, let's start collaborating. We have recently moved the repo from spotify org to backstage org - all links look good.